lib/html/template/rose-bush/view.html

```diff
@@ -27,39 +27,36 @@
 {% include "job-entry.html" %}
 </div>
 {% endif -%}
-</div>
 
+<br />
+
+<!-- File Search -->
 <div class="container-fluid">
-<div class="row-fluid">
-<div class="span1 text-right">
-<pre class="prettyprint">
-{% for i in range(1, lines|length + 1) -%}
-<span><a id="{{i}}" href="#{{i}}">{{i}}</a></span>
-{% endfor -%}
-</pre>
+  <div class="form-inline">
+    <div class="input-group stylish-input-group">
+	<form name="file_search" onsubmit="return FileLoader.fileSearch();">
```

Comment:
I don't know about the latest standard, but I think `<form>` used to require an `action=` attribute. You can then do `action="javascript:FileLoader.fileSearch()"` instead of using an `onsubmit=` attribute.